### PR TITLE
CDAP-1707: Fix AltStyleTableRenderer when there are long headers

### PR DESCRIPTION
**Pull request by jwang47**
https://issues.cask.co/browse/CDAP-1707

### CODE REVIEW

**Comment by jwang47** [comment]
http://builds.cask.co/browse/CDAP-DUT999

File: cdap-cli/src/test/java/co/cask/cdap/cli/util/table/TableRendererTest.java [hunk 1]
```diff
@@ -78,4 +78,17 @@ public void testTwoLineCell2() {
       .build();
     getRenderer().render(System.out, table);
   }
+
+  @Test
+  public void testLongHeader() {
```
Comment (jguery) [inline cdap-cli/src/test/java/co/cask/cdap/cli/util/table/TableRendererTest.java]: How do we actually make sure that the header stays in the table? I don't see any assertions? Is it just a visual test?

**Comment by jguery** [comment]
One comment, otherwise LGTM.

File: cdap-cli/src/test/java/co/cask/cdap/cli/util/table/TableRendererTest.java (hunk 1 above)
Comment (jwang47) [inline cdap-cli/src/test/java/co/cask/cdap/cli/util/table/TableRendererTest.java]: Yeah, this test is not meant to be a real test.

Comment (jguery) [inline cdap-cli/src/test/java/co/cask/cdap/cli/util/table/TableRendererTest.java]: Ok, that's fine then, if all the tests of that class are like that.

**Comment by jwang47** [comment]
All tests passed except for tests in one unrelated module.

**Pull request merged**